RELEASING.md

```diff
@@ -26,3 +26,54 @@
         helm repo update elastic
         helm search repo -l elastic/apm-attacher
         ```
+
+## Testing procedure (for Linux, Windows works too but you need to adjust how files are created or create them manually)
+
+1. Clone (or update) the repo locally
+    - `git clone https://github.com/elastic/apm-k8s-attacher.git`
+2. `cd apm-k8s-attacher`
+3. Create the custom values ymal file - replacing the secret token and url with valid values for a server is better, buy even with these dummy values testing still works, just the agent won't connect to a server
+```
+cat > custom.yaml <<EOF
+apm:
+  secret_token: SuP3RT0K3N 
+  namespaces: 
+    - default
+webhookConfig:
+  agents:
+    java: 
+      environment:
+        ELASTIC_APM_SERVER_URL: "https://apm-example.com:8200" 
+        ELASTIC_APM_ENVIRONMENT: "prod"
+        ELASTIC_APM_LOG_LEVEL: "info"
+EOF
+```
+4. Install the attacher - note the namespace can be any namespace but can't be the default nor a namespace where pods will be tested
+    - `helm install test-main charts/apm-attacher --values custom.yaml --namespace=elastic-apm --create-namespace` 
+5. Create a pod to test - the example here is a Java pod which uses a known image that holds a testing app
+```
+cat > test-app.yaml <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-app
+  annotations:
+    co.elastic.apm/attach: java 
+  labels:
+    app: test-app
+spec:
+  containers:
+    - image: docker.elastic.co/demos/apm/k8s-webhook-test
+      imagePullPolicy: Always
+      name: test-app
+      env: 
+      - name: ELASTIC_APM_TRACE_METHODS
+        value: "test.Testing#methodB"
+EOF
+```
+6. Start the app and check the logs
+    - `kubectl apply -f test-app.yaml`
+    - `kubectl logs test-app`
```

Comment:
Perhaps could state what one needs to look for in the logs to confirm it is working. Presumably this is looking for the preamble log message from the Java agent.

Reply:
good point

Reply:
```suggestion
    - `kubectl logs test-app`
    - and look for a log line that shows the agent has started, eg `co.elastic.apm.agent.configuration.StartupInfo - Starting Elastic APM`
```